companion/src/mdichild.cpp

```diff
@@ -65,9 +90,12 @@ MdiChild::~MdiChild()
   delete ui;
 }
 
-void MdiChild::eepromInterfaceChanged()
+void MdiChild::refresh(bool expand)
 {
-  ui->modelsList->refreshList();
+  modelsListModel->refresh();
+  if (1 || expand) {
```

Comment:
yes, temp code, I don't like that the refresh method clears and fills again the list. until this is better we have to always expand the list